src/main/java/seedu/duke/template/RcTemplate.java

```diff
@@ -0,0 +1,56 @@
+package seedu.duke.template;
+
+import seedu.duke.component.Capacitor;
+import seedu.duke.component.Resistor;
+
+
+public class RcTemplate extends Template {
+    private static final String RC_TEMPLATE = "\t+---R-----C---+\n"
+            + "\t|             |\n"
+            + "\t|             |\n"
+            + "\t+----+V_ac+---+\n";
+
+    private Resistor resistor;
+    private Capacitor capacitor;
```

Comment:
Could consider having setters for these? @sevenseasofbri `LrTemplate` could use them as well?

Reply:
Yup was thinking of setters, wasn't sure whether I should add. Will add after tdys meeting!